openpaas-james/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/openpaas/james/common/LinagoraKeystoreSetMethodContract.scala

```diff
@@ -63,10 +111,10 @@ trait LinagoraKeystoreSetMethodContract {
          |		[
          |			"Keystore/set",
          |			{
-         |				"accountId": "$generateAccountIdAsString",
+         |				"accountId": "$ACCOUNT_ID",
          |				"created": {
-         |          "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6": {
-         |            "id": "1"
+         |          "K87": {
+         |            "id": "$PGP_KEY_ID"
```

Comment:
I put a hardcoded key id now. I saw it was generated same keyid for same key in memory. I would not be surprised that the algorithm might be different for Cassandra key id? Not sure of the approach here

Reply:
> Cassandra key id?

It will use the key fingerprint as well.